libs/sdk-backend-tiger/src/utils/api.ts

```diff
@@ -1,8 +1,19 @@
-// (C) 2019-2020 GoodData Corporation
+// (C) 2019-2021 GoodData Corporation
 import { isUriRef, ObjRef, Uri, isIdentifierRef } from "@gooddata/sdk-model";
+import sdk from "@gooddata/api-client-tiger";
 import { TigerAuthenticatedCallGuard } from "../types";
 import { UnexpectedError } from "@gooddata/sdk-backend-spi";
 
+/**
+ * Returns organization name
+ *
+ * @public
+ */
+export const getOrganizationTitle = async (): Promise<string> => {
+    const data = (await sdk.axios.get("/api/entities/organization")).data;
```

Comment:
Please add a TODO to replace this once the generated client supports this (we ideally do not want any URLs in the sdk-backend-* code).